internal/usecase/interfaces/user.go

```diff
@@ -44,6 +44,8 @@ type UpdateMeParam struct {
 type User interface {
 	Fetch(context.Context, []id.UserID, *usecase.Operator) ([]*user.User, error)
 	Signup(context.Context, SignupParam) (*user.User, *user.Team, error)
+	CreateVerification(context.Context, string) (string, error)
```

Comment:
```suggestion
	CreateVerification(context.Context, string) error
```

string return arg is not used. because it is enough to detect whether the sending email is succeeded by checking error is nil